transport/http/handler.go

```diff
@@ -147,13 +149,11 @@ func handleOnewayRequest(
 	return nil
 }
 
-func updateSpanWithErr(span opentracing.Span, err error) error {
+func updateSpanWithErr(span opentracing.Span, err error) {
```

Comment:
I would change this back.

The return value was there for convenience so you could always do,

    return updateSpanWithErr(span, err)

Regardless of whether there was an error or not instead of,

    if err != nil {
        updateSpanWithErr(span, err)
    }
    return err